src/controllers/deliveryLog.js

```diff
@@ -1,24 +1,32 @@
 import { sendDataResponse } from '../utils/responses.js'
+import DeliveryLog from '../domain/deliveryLog.js'
+import DeliveryLogLines from '../domain/deliveryLogLines.js'
 
 export const create = async (req, res) => {
   const { date, cohort_id: cohortId, lines } = req.body
+  const userId = req.user.id
+  const newDeliveryLogData = {
+    date: new Date(date),
+    cohortId,
+    userId
+  }
 
-  return sendDataResponse(res, 201, {
-    log: {
-      id: 1,
-      cohort_id: cohortId,
-      date,
-      author: {
-        id: req.user.id,
-        first_name: req.user.firstName,
-        last_name: req.user.lastName
-      },
-      lines: lines.map((line, index) => {
-        return {
-          id: index + 1,
-          content: line.content
-        }
-      })
-    }
-  })
+  const deliveryLogToCreate = await DeliveryLog.fromJson(newDeliveryLogData)
+  const deliveryLog = await deliveryLogToCreate.save()
+
+  const deliveryLogLines = []
+  const postId = deliveryLog.id
```

Comment:
1. rename this variable `postId`  to `deliveryLogId` here and where it's used
2. Or delete it -> seems unnecessary to store this data in a variable

Reply:
Removed postId variable.